lib/iris/fileformats/ugrid_cf_reader.py

```diff
@@ -188,8 +212,23 @@ def complete_ugrid_cube(self, cube):
             raise ValueError(msg.format(meshes_info))
         if meshes_info:
             i_dim, (mesh, mesh_location) = meshes_info[0]
+            mesh_var = self.dataset.variables[mesh.mesh_name]
+
+            topology_dimension = mesh_var.getncattr("topology_dimension")
+            node_coordinates = []
+            for node_var_name in mesh_var.getncattr("node_coordinates").split(
+                " "
+            ):
+                node_var = self.dataset.variables[node_var_name]
+                node_coordinates.append(node_var.getncattr("standard_name"))
```

Comment:
I can't help feeling we should be applying the same here as in Iris --> standard_name or long_name or var_name.